task_12/index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <title>Pizza App</title>
+  </head>
+  <body>
+    <header class="flex-container header">
+      <div class="lbl">
+        <i class="fa fa-clock-o fa-fw label" aria-hidden="true"></i>
+        <time id="clock">00:00:00</time>
```

Comment:
Consider adding `datetime` attribute for improved accessibility.

Reply:
Thanks for the advice. Added.